3.2/Dockerfile.fedora

```diff
@@ -13,6 +13,7 @@ ENV NAME=redis \
     ARCH=x86_64
 
 ENV REDIS_VERSION=$VERSION \
+    APP_DATA=${APP_ROOT}/src \
```

Comment:
Already defined in `s2i-core` as `HOME/` (https://github.com/sclorg/s2i-base-container/blob/master/core/Dockerfile#L28)

Reply:
I knew about it but wasn't sure whether I could rely on that, mariadb also uses $APP_DATA

Reply:
mariadb changes value of HOME therefore APP_DATA is needed.

Although the commend in `core` Dockerfile is quite misleading for me:
`# The $HOME is not set by default, but some applications needs this variable`
It doesn't seem like some our requirement. So if you prefer APP_DATA I have no problem with it:-)

Reply:
Yes, comment in core is a bit misleading, but I'll leave $APP_DATA here because it shows up in other images and I think the meaning is correct. We'll discuss tomorrow. :-)